### ISSUE TITLE
Connected to Metamask but not created the Session

### ISSUE THREAD

**Issue by gowtham-calib** [WalletConnect/WalletConnectSwiftV2#996]
**Describe the bug**
I have created the URI from the Pair. instance.create() and generated the QR code for the URI and connected with the metamask.
It shows the wallet connected successfully but when I tried to check the wallet connect sessions in the metamask, it shows empty 
No Session is created.

Also need to know, how to check the session details from the socket.

**SDK Version**
 - Client: Swift
 - Swift Verison: 5
 - iOS Version: 14
 
**To Reproduce**
`let metadata = AppMetadata(
            name: "Example Wallet",
            description: "Wallet description",
            url: "example.wallet",
            icons: ["https://avatars.githubusercontent.com/u/37784886"]
        )
        Pair.configure(metadata: metadata)
        let uri = try await Pair.instance.create()
        
        let namespaces: [String: ProposalNamespace] = [
            "eip155": ProposalNamespace(
                chains: [
                    Blockchain("eip155:80001")!,
                ],
                methods: [
                    "eth_sendTransaction",
                    "personal_sign",
                    "eth_signTypedData"
                ], events: []
            ),
        ]
        
    try await Sign.instance.connect(requiredNamespaces: namespaces, topic: uri.topic)
            
            url = uri.absoluteString
            QRCodeImgView.image = generateQRCode(from: uri.absoluteString)
            `

**Expected behavior**
Need to update the session in the wallet and need to check session details from the socket. 

**Screenshots**
![image](https://github.com/WalletConnect/WalletConnectSwiftV2/assets/82747133/f75f70a1-e702-4ebc-b5fd-ff2d6bb2eb05)
![image](https://github.com/WalletConnect/WalletConnectSwiftV2/assets/82747133/eb63a313-df6b-4bed-9e0c-de1df88037fc)

**Comment by reaperdtme** [comment]
Seeing the same issue.

**Comment by sxiaojian88** [comment]
I face the same issue. No Session is created.
SDK Version

Client: Swift
Swift Verison: 5
iOS Version: 15
To Reproduce
Steps to reproduce the behavior:
`


        let methods: Set<String> = ["eth_sendTransaction", "personal_sign"]
        let blockchains: Set<Blockchain> = [Blockchain("eip155:1")!]
        let namespaces: [String: ProposalNamespace] = [
            "eip155": ProposalNamespace(
                chains: blockchains,
                methods: methods,
                events: []
            )
        ]

        let defaultSessionParams =  SessionParams(
                                        requiredNamespaces: namespaces,
                                        optionalNamespaces: nil,
                                        sessionProperties: nil
                                    )
        WalletConnectModal.configure(
            projectId: "xxxx",
            metadata: metadata,
            sessionParams: defaultSessionParams
        )
`

**Comment by PetecOvod** [comment]
Same issue. ETA for fix?

**Comment by llbartekll** [comment]
@alexander-lsvk can you take a look?

**Comment by PetecOvod** [comment]
Any update?

**Comment by willis-tagheuer** [comment]
Duplicate issue : https://github.com/WalletConnect/WalletConnectSwiftV2/issues/981

**Comment by PetecOvod** [comment]
Same result ((

**Comment by PetecOvod** [comment]
Yes. There is a session in MetaMask but WalletConnect doesn't provide session back in sample app after confirm connection on the MetaMask app.  Like in https://github.com/WalletConnect/WalletConnectSwiftV2/issues/981
